tests/functional/c/consider/consider_using_with_open.py

```diff
@@ -52,17 +55,22 @@ def test_open_inside_with_block():
 
 def test_ternary_if_in_with_block(file1, file2, which):
     """Regression test for issue #4676 (false positive)"""
-    with (open(file1, encoding="utf-8") if which else open(file2, encoding="utf-8")) as input_file:  # must not trigger
+    with (
+        open(file1, encoding="utf-8") if which else open(file2, encoding="utf-8")
+    ) as input_file:  # must not trigger
         return input_file.read()
 
 
 def test_single_line_with(file1):
-    with open(file1, encoding="utf-8"): return file1.read()  # must not trigger
+    with open(file1, encoding="utf-8"):
+        return file1.read()  # must not trigger
```

Comment:
Would you mind reverting the unrelated formatting changes? As you can tell from the test method names, they're testing edge cases relating to oddly formatted code. We don't want to unintentionally degrade test coverage.